.idea/runConfigurations/Firestore_Integration_Tests__emulator_.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="Firestore Integration Tests" type="AndroidTestRunConfigurationType" factoryName="Android Instrumented Tests">
+  <configuration default="false" name="Firestore Integration Tests (emulator)" type="AndroidTestRunConfigurationType" factoryName="Android Instrumented Tests">
```

Comment:
Can we be explicit here that the emulator being referred to here is the backend emulator? The term is overloaded in this context :)